src/components/App.tsx

```diff
@@ -35,24 +34,25 @@ function App() {
           </Link>
           .
         </Alert>
+        {/* What would we like to do with this feedback box ?*/}
         <Box display='flex' justifyContent='flex-end' alignItems='flex-end'>
           <FeedbackAlert />
-          <ToggleDarkMode
-            toggleColorMode={toggleColorMode}
-            theme={protocolTheme}
-          />
         </Box>
+        <PerfCompareHeader
```

Comment:
From your note "I moved the header to the App file because it's a constant on every page; only one page requires an update on the header which can easily be changed.". -> Only the PerfCompare logo is constant on Search View and Results View. Did you take this into consideration when moving `PerfCompareHeader`?

Reply:
Perfcompare header imported only in Search and Results view; removed import from App